scripts/packageAdBlock.js

```diff
@@ -59,14 +68,30 @@ const processComponent = (binary, endpoint, region, keyDir,
   const parsedManifest = util.parseManifest(originalManifest)
   const id = util.getIDFromBase64PublicKey(parsedManifest.key)
 
+  let contentHash
+  if (id === regionalCatalogComponentId) {
+    const contentFile = path.join('build', 'ad-block-updater', componentSubdir, 'regional_catalog.json')
+    contentHash = util.generateSHA256HashOfFile(contentFile)
+  } else if (id === resourcesComponentId) {
+    const contentFile = path.join('build', 'ad-block-updater', componentSubdir, 'resources.json')
+    contentHash = util.generateSHA256HashOfFile(contentFile)
+  } else if (id.length === 32) {
+    const contentFile = path.join('build', 'ad-block-updater', componentSubdir, 'list.txt')
+    contentHash = util.generateSHA256HashOfFile(contentFile)
+  }
+
```

Comment:
Nit: Since these branches are all essentially doing the same thing, maybe they can be refactored a bit or extracted into a function?

Reply:
yep, good call! I updated this as well